### PR TITLE
Adjust default settings for better user experience

### PR DESCRIPTION
**Pull request by MichaelOrlov**
Add modified colour scheme, adjust size of default font, show toolbar by default, increase buffer for terminal, set apply non-conflicted changes during merge, set default keymap to VS.

---
This change is [Reviewable](https://reviewable.io/reviews/apexai/ros2_clion_style/13)

### CODE REVIEW

**Comment by jpsamper2009** [comment]
@MichaelOrlov What about having another Apex.OS developer review this? I don't really use CLion on a day-to-day basis so I can't really comment on the user experience...

**Comment by MichaelOrlov** [comment]
@sumanth-nirmal Wouldn't you mind to review this pull request?

File: find.xml [hunk 1]
```diff
@@ -3,5 +3,30 @@
...
+    <mask>*.hp</mask>
+    <mask>*.hxx</mask>
+    <mask>*.ii</mask>
+    <mask>*.tcc</mask>
+    <mask>*.mm</mask>
+    <mask>*.cc</mask>
+    <mask>*.h</mask>
+    <mask>*.c</mask>
+    <mask>*.hpp</mask>
+    <mask>*.cpp</mask>
```
Comment (sumanth-nirmal) [inline find.xml:30]: @MichaelOrlov Will it now searches in all these files by default?

Comment (MichaelOrlov) [reply find.xml:30]: @sumanth-nirmal Nop. by default it searches in any files. \
But if you will chose `File mask:` checkbox. List of this masks will be available for selection.

**Review by sumanth-nirmal — COMMENTED** [review]
> Reviewed 9 of 9 files at r1.
> *[Reviewable](https://reviewable.io/reviews/apexai/ros2_clion_style/13)* status: all files reviewed, 1 unresolved discussion (waiting on @sumanth-nirmal)

**Review by sumanth-nirmal — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/apexai/ros2_clion_style/13#-:-MBSNXxA2xItWJMpYBeJ:bg7igk9)* status: all files reviewed, 1 unresolved discussion (waiting on @sumanth-nirmal)

**Review by sumanth-nirmal — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/apexai/ros2_clion_style/13)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**